src/spritekit.cs

```diff
@@ -402,6 +403,7 @@ partial interface SKNode : NSSecureCoding, NSCopying {
 	}
 
 #if MONOMAC
```

Comment:
Why not removing the #if MONOMAC?

Reply:
Good question. Let me inspect

Reply:
@mandel-macaque They were not necessary! They have been removed!